EIPS/eip-1949.md

```diff
@@ -0,0 +1,139 @@
+---
+eip: 1949
+title: Delayed Distributed Minting
+author: Johann Barbie <@johannbarbie>, Pinkiebell <@pinkiebell>
+discussions-to: https://ethresear.ch/t/a-distributed-breeding-function/5264
+status: Draft
+type: Standards Track
+category: ERC
+created: 2019-04-18
+requires: ERC721
+requires: ERC1948
+---
+
+## Simple Summary
+
+```
+Two CryptoKitties take a honeymoon on a side-chain 🐈🐈🏖. The blazing fast transactions heat their blood 🔥, one thing leads to another ❤ and a rare baby-kitten is born 💥🐯. On the way home to mainnet the family gets stopped at the bridge contract, “👮‍♂️ tokenIDs please!”. Oh shoot, the baby-kitten doesn’t have one on the root chain 🤷‍♀️.
+```
+
+The minting function of NFT contracts is centralized to a single chain. This proposal allows to mint tokens across multiple chains.
+
+## Abstract
+
+This ERC introduced a design for NFT tokens to be mintable across multiple chains. A minimal requirement for distributed minting are collision-free token Ids. The Collision-freeness is achieved by introducing to types of tokens, queens and workers. The Ids of the queens are generated by hashing the token contract address with an increasing `queenCounter` in the contract. The Ids of the workers are generated by hashing the queenIds with an increasing counter that the queen tokens carry in themselves using the [data token specification](https://github.com/ethereum/EIPs/pull/1948).
```

Comment:
I thin `collision-freeness` should be lowercase. Also likely in `introducing to types` you meant `introducing two types`.